common/types/shared.go

```diff
@@ -25,12 +25,44 @@ type AccessDeniedError struct {
 	Message string
 }
 
+func (v *AccessDeniedError) GetMessage() (o string) {
+	if v != nil {
+		return v.Message
+	}
+	return
+}
+
+// ActivityLocalDispatchInfo is an internal type (TBD...)
+type ActivityLocalDispatchInfo struct {
+	ActivityID *string
```

Comment:
please add the all fields to match the original struct https://github.com/uber/cadence/blob/6e5f9ece2d632d070a3ad5640dcdc7d90abb4f7d/.gen/go/shared/shared.go#L179:

```
type ActivityLocalDispatchInfo struct {
	ActivityId                      *string `json:"activityId,omitempty"`
	ScheduledTimestamp              *int64  `json:"scheduledTimestamp,omitempty"`
	StartedTimestamp                *int64  `json:"startedTimestamp,omitempty"`
	ScheduledTimestampOfThisAttempt *int64  `json:"scheduledTimestampOfThisAttempt,omitempty"`
	TaskToken                       []byte  `json:"taskToken,omitempty"`
}
```